README.md

```diff
@@ -10,19 +10,39 @@ The target user of this repository is package maintainers who want to build thei
 Each release in this repository contains a tip-of-tree build of Pyodide of the given date.
 You can use the `pyodide xbuildenv install` command to install the build environment for a given date.
 
+### Cross build environments
+
+#### Release builds
+
 ```bash
 pip install pyodide-build
 
 # Change the date to the date of the build you want to use
-pyodide xbuildenv install --url "https://github.com/pyodide/pyodide-build-environment-nightly/releases/download/20250125/xbuildenv.tar.bz"
+pyodide xbuildenv install --url "https://github.com/pyodide/pyodide-build-environment-nightly/releases/download/20250125/xbuildenv.tar.bz2"
 
 # Now you can use the build environment to build your package
 pyodide build
 ```
 
+#### Debug builds
+
+For debugging purposes, you can also use the debug build environment:
+
+```bash
+pip install pyodide-build
+
+# Change the date to the date of the build you want to use
+pyodide xbuildenv install --url "https://github.com/pyodide/pyodide-build-environment-nightly/releases/download/20250125/xbuildenv-debug.tar.bz2"
+
+# Now you can use the debug build environment to build your package
+pyodide build
```

Comment:
Actually, it is worth noting that using the debug build does not make any difference during the build step.

It is only useful when you use `pyodide venv` with that xbuildenv, or when you copy that debug runtime and use it in pytest.